packages/core/src/CBOR/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './TxCBOR';
+export * from './TxBodyCBOR';
```

Comment:
Could extract all `core` features to a separate commit

Reply:
Agree. It needs some unit test coverage too

Reply:
2fcc3f6806e0638ede2095d4d180e262d9160f43 and did a small refactor for an inconsistency that I noticed in the existing test 21daef2f50cd6b05bc74f1bdc21a36ef4a361657